### DISCUSSION TITLE
Testing a Mura 7.1 to Masa update

### DISCUSSION THREAD

**Discussion by brimard** [MasaCMS/MasaCMS#148] [General]
Is the process for updating Mura 7.1 to Masa still the same nowadays as outlined on https://www.masacms.com/docs/?

-     Go to your settings.ini.cfm file
-     Update the property autoupdateurl to point to: https://codeload.github.com/MasaCMS/MasaCMS/zip/main
-     Reload MuraTM using the 'Reload Application' option in the left menu to pick up the new setting
-     Use the MuraTM 'Update core' function
-     Enjoy MASA CMS!

**Comment by guustnieuwenhuis** [comment]
For version 7.1+, yes.

**Comment by withLogic** [comment]
Having never gone through a MuraCMS update. We are on Mura 7.1 and want to move to the latest MasaCMS. I have a few questions:
1. Is there a process or documentation on best practices?
2. What are the major risks involved?
3. Approximately how long does the update take? 

Thanks!

**Comment by guustnieuwenhuis** [comment]
1. Yes: https://docs.masacms.com/getting-started/upgrade-from-mura-to-masa-cms/
2. If you're already on 7.1, using the latest Mura folder structure: no major risks. If you're on an older version of Mura, you will have to execute the upgrade manually and take this information in account: https://docs.murasoftware.com/v7-1/installation-setup/upgrading-mura/
3. A minute or 2.

Make sure you execute the upgrade on a dev or test environment first and test. This isn't Masa CMS specific, but a general practice in software development. :)
